.gitignore

```diff
@@ -12,8 +12,7 @@ gradle-app.setting
 .gradletasknamecache
 
 # # Work around https://youtrack.jetbrains.com/issue/IDEA-116898
-# gradle/wrapper/gradle-wrapper.properties
-
+gradle/wrapper/gradle-wrapper.properties
```

Comment:
Okay, I asked for this to be commented back out, and while the deleted file is back, this is still not commented out.